test/test_images/probe_helper/utils/k8s_client.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2021 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
+
+import "k8s.io/client-go/kubernetes"
+
+type K8sClient struct {
```

Comment:
Do we need this type (& this file)? Why can't we just use kubernetes.Interface directly?

Reply:
We don't -- I originally created this to solve a problem that's no longer relevant. Will remove.